cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/MockETLHandler.java

```diff
@@ -63,8 +63,23 @@ public void getTemplates(HttpRequest request, HttpResponder responder) throws Ex
   public void getTemplate(HttpRequest request, HttpResponder responder, @PathParam("template-id") String templateId)
     throws Exception {
     JsonObject template = new JsonObject();
+    JsonArray properties = new JsonArray();
     template.addProperty("name", templateId);
     template.addProperty("description", "Template used to create ETL Adapters");
+    if (templateId.equalsIgnoreCase(BATCH)) {
```

Comment:
There might be many including user created ones. This is just a mock implementation so that UI can test the APIs etc.